### PR TITLE
Add requirement for Token authentication on optout endpoint

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

File: casepro/backend/junebug.py [hunk 1]
```diff
@@ -438,6 +440,26 @@ def get_url_patterns(self):
         ]
 
 
+def token_auth_required(auth_token_func):
```
Comment (hodgestar) [inline casepro/backend/junebug.py]: We should add a docstring for this so people know what is does. :)

**Comment by hodgestar** [comment]
Other than the request for a docstring, looks good to me.

Aside: We do need to consolidate the Junebug backend settings into a single settings dictionary though, I think (not this PR though).

**Comment by KaitCrawford** [comment]
@hodgestar ready for review

**Comment by hodgestar** [comment]
:+1:

**Comment by KaitCrawford** [comment]
PR sent straight to Casepro here https://github.com/rapidpro/casepro/pull/116

**Pull request closed without merging**